posthog-node/test/posthog-node.spec.ts

```diff
@@ -919,6 +924,34 @@ describe('PostHog Node.js', () => {
       )
       mockedFetch.mockClear()
 
+      expect(
+        await posthog.getFeatureFlagPayload('beta-feature', 'some-distinct-id', undefined, {
+          personProperties: { region: 'USA', name: 'Aloha' },
+        })
+      ).toEqual({ variant: 'A' })
+
+      // TRICKY: There's now an extra step before events are queued, so need to wait for that to resolve
+      jest.runOnlyPendingTimers()
+      await waitForPromises()
+      await posthog.flush()
+
+      expect(mockedFetch).toHaveBeenCalledWith('http://example.com/batch/', expect.any(Object))
+
+      expect(getLastBatchEvents()?.[0]).toEqual(
+        expect.objectContaining({
+          distinct_id: 'some-distinct-id',
+          event: '$feature_flag_payload_called',
+          properties: expect.objectContaining({
+            $feature_flag: 'beta-feature',
+            $feature_flag_response: true,
+            $feature_flag_payload: { variant: 'A' },
+            locally_evaluated: true,
+            [`$feature/${'beta-feature'}`]: true,
+          }),
+        })
+      )
+      mockedFetch.mockClear()
```

Comment:
added a test case that the `getFeatureFlagPayload` method also sends a `feature_flag_called` event.